en/docs/develop/oidc-scope-management-rest-apis.md

```diff
@@ -3,14 +3,20 @@ template: templates/swagger.html
 ---
 # OpenID Connect Scope Management Rest API Definition - v1
 
-??? Note "Note: Click to view"
-    Do the following to try out the REST APIs with your local instance of WSO2 Identity Server.
-
-    1.  Expand the relevant API operation and click the **Try It Out** button.
-    2.  Fill in relevant sample values for the input parameters and click **Execute**.
-        You will receive a sample curl command with the sample values you filled in.
-    3. Add a `-k` header to the curl command and run the curl command on the terminal with a running instance of WSO2 IS.
+The OIDC scope API in WSO2 Identity Server can be used to manage OIDC scopes and scope claim mapping. OIDC scope is a
```

Comment:
WSO2 Identity Server(IS)
Since OIDC scope is a sub category of OAuth2 scopes, these end points cannot have the same scope names in WSO2 IS. See [OAUTH2 Scope endpoint]() for further information.